lib/containers/container_images.pm

```diff
@@ -82,7 +82,7 @@ sub build_and_run_image {
     assert_script_run("$runtime logs myapp");    # show logs for easier problem investigation
 
     # Test that the exported port is reachable
-    my $curl_opts = "--retry 50";
+    my $curl_opts = "--retry " . get_var('FIPS_ENABLED') ? "500" : "100";
```

Comment:
Are we sure that retry 500 times will help to pass ?

Reply:
> Are we sure that retry 500 times will help to pass ?

I included a verification run.

Reply:
VR passing is not a description of the logic leading to the change

Reply:
> it could be useful to add also `--retry-max-time <seconds>` or `--max-time <seconds>`

We already have the default timeout of `assert_script_run`.  I prefer to specify the number of retries than specifying timeouts, which is always prone to errors.